src/Kanvas/Companies/Branches/Repositories/CompaniesBranches.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Companies\Branches\Repositories;
+
+use Kanvas\Companies\Models\CompaniesBranches as CompaniesBranchesModel;
+
+class CompaniesBranches
```

Comment:
@FredPeal this is not needed use UserRepository::belongsToCompanyBranch , for this specific function